### ISSUE TITLE
Buttons must have discernible text

### ISSUE THREAD

**Issue by nhjohnstone** [web-illinois/ilw-header-menu#15]
This is getting flagged in Axe DevTools

`<button class="arrow-only" aria-controls="items" aria-expanded="false">`

I'm not sure the way to go here, but Axe is suggesting:

- Element does not have inner text that is visible to screen readers
- aria-label attribute does not exist or is empty

**Comment by dandalpiaz** [comment]
I ran into this as well. When using a screen reader, when the arrow drop-down button is focused the only think that is announced is 'button, collapsed'. This appears to be an issue with the way the `linked="true"` attribute is building the link and button separately. I can try to fix with a PR. 

Sample page showing issue:

<img width="510" height="181" alt="menu with linked menu section" src="https://github.com/user-attachments/assets/5d31d72f-c21a-4b11-865a-bcd8a633d55d" />

```html
<!DOCTYPE html> 
<html lang="en"> 
<head> 
    <meta charset="utf-8">
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> 
    <meta name="viewport" content="width=device-width, initial-scale=1"> 
    <link rel="stylesheet" href="https://cdn.toolkit.illinois.edu/3/toolkit.css">
    <script type="module" src="https://cdn.toolkit.illinois.edu/3/toolkit.js"></script>
</head>
<body style="margin: 0;">
<ilw-page>
<ilw-header slot="header">
    <ilw-header-menu slot="navigation">
        <ul>
            <li>
                <ilw-header-menu-section linked="true">
                    <a slot="link" href="/philosophy/index.html">Information</a>
                    <ul>
                        <li><a href="/philosophy/index.html">Philosophy</a></li>
                        <li><a href="/links/index.html">Helpful Links</a></li>
                        <li><a href="/github/index.html">Organization and Github</a></li>
                    </ul>
                </ilw-header-menu-section>
            </li>
        </ul>
    </ilw-header-menu>
</ilw-header>
</ilw-page>
</body>
</html>
```
